frontend/src/components/pages/Scheduling/index.tsx

```diff
@@ -48,7 +48,7 @@ const schedulingDefaultData = ({
   dayPart: "",
   startTime: "",
   endTime: "",
-  volunteerIds: [],
+  volunteerId: "",
```

Comment:
we can actually remove this because volunteerId is not provided when scheduling a donation and if I try to schedule a donation right now, this will throw an error that volunteerId is not an integer since we are passing an empty string.